From: Justin I. Nevill <jnevill@redhat.com>
Date: Thu, 8 Sep 2011 15:36:38 -0400
Subject: [PATCH] Fix error from "grub-install /dev/cciss/c0d0"

This patch fixes the error generated on the invocation of
"grub-install /dev/cciss/c0d0".

---
 grub-install.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/grub-install.in b/util/grub-install.in
--- a/util/grub-install.in	2011-09-08 10:47:12.410508634 -0400
+++ b/util/grub-install.in	2011-09-08 15:36:38.570972684 -0400
@@ -117,7 +117,7 @@
 				  -e 's%.*/fd[0-9]*$%%' \
 				  -e 's%.*/floppy/[0-9]*$%%' \
 				  -e 's%.*/\(disc\|part\([0-9]*\)\)$%\2%' \
-				  -e 's%.*c[0-7]d[0-9]*p%%' \
+				  -e 's%.*c[0-7]d[0-9]*%%' \
 	  -e 's%.*/mapper/[[:alpha:]]\+[[:digit:]]\+p\([[:digit:]]\+\)$%\1%' \
 	  -e 's%.*/mapper/isw_[[:alpha:]_]\+[[:alpha:]]\+[[:digit:]]\+p\([[:digit:]]\+\)$%\1%' \
 	  -e 's%.*/mapper/[[:alpha:]]\+_[[:alpha:]]\+\([[:digit:]]\+\)$%\1%' |
